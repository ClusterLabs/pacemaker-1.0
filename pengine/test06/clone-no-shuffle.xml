 <cib generated="true" admin_epoch="0" epoch="5" num_updates="41" ignore_dtd="false" num_peers="2" have_quorum="1" cib_feature_revision="2.0" crm_feature_set="2.1" dc_uuid="6676c8d5-552e-4a86-a2ea-172e954f7342" ccm_transition="4" cib-last-written="Wed Mar 26 11:55:00 2008">
   <configuration>
     <crm_config>
       <cluster_property_set id="cib-bootstrap-options">
         <attributes>
           <nvpair id="cib-bootstrap-options-dc-version" name="dc-version" value="0.6.2-node: 7f22385d1b41c36c4cc54f292e55869caa94ab7e"/>
           <nvpair name="no-quorum-policy" value="stop" id="no-quorum-policy"/>
           <nvpair name="symmetric-cluster" value="true" id="symmetric-cluster"/>
           <nvpair name="stonith-enabled" value="true" id="stonith-enabled"/>
           <nvpair name="stonith-action" value="reboot" id="stonith-action"/>
           <nvpair name="default-resource-stickiness" value="0" id="default-resource-stickiness"/>
           <nvpair name="default-resource-failure-stickiness" value="-INFINITY" id="default-migration-threshold"/>
           <nvpair name="is-managed-default" value="true" id="is-managed-default"/>
           <nvpair name="default-action-timeout" value="20s" id="default-action-timeout"/>
           <nvpair name="stop-orphan-resources" value="true" id="stop-orphan-resources"/>
           <nvpair name="stop-orphan-actions" value="true" id="stop-orphan-actions"/>
           <nvpair name="remove-after-stop" value="false" id="remove-after-stop"/>
           <nvpair name="startup-fencing" value="true" id="startup-fencing"/>
           <nvpair name="last-lrm-refresh" id="cib-bootstrap-options-last-lrm-refresh" value="1204805697"/>
         </attributes>
       </cluster_property_set>
     </crm_config>
     <nodes>
       <node id="6676c8d5-552e-4a86-a2ea-172e954f7342" uname="dktest2sles10" type="normal"/>
       <node id="c563f60c-45a8-439e-85dc-645f4ecc150d" uname="dktest1sles10" type="normal"/>
     </nodes>
     <resources>
       <master_slave id="ms-drbd1">
         <meta_attributes id="ma-ms-drbd1">
           <attributes>
             <nvpair id="ma-ms-drbd1-1" name="clone_max" value="2"/>
             <nvpair id="ma-ms-drbd1-2" name="clone_node_max" value="1"/>
             <nvpair id="ma-ms-drbd1-3" name="master_max" value="1"/>
             <nvpair id="ma-ms-drbd1-4" name="master_node_max" value="1"/>
             <nvpair id="ma-ms-drbd1-5" name="notify" value="yes"/>
             <nvpair id="ma-ms-drbd1-6" name="globally_unique" value="false"/>
             <nvpair id="ma-ms-drbd1-7" name="target_role" value="started"/>
           </attributes>
         </meta_attributes>
         <primitive class="ocf" provider="heartbeat" type="drbd" id="drbd1">
           <instance_attributes id="ia-drbd1">
             <attributes>
               <nvpair id="ia-drbd1-1" name="drbd_resource" value="drbd1"/>
             </attributes>
           </instance_attributes>
           <operations>
             <op id="op-ms-drbd1-1" name="monitor" interval="10s" timeout="15s" start_delay="30s" role="Master"/>
             <op id="op-ms-drbd1-2" name="monitor" interval="11s" timeout="15s" start_delay="30s" role="Slave"/>
           </operations>
         </primitive>
       </master_slave>
       <primitive id="testip" class="ocf" type="IPaddr2" provider="heartbeat">
         <instance_attributes id="ia-testip">
           <attributes>
             <nvpair id="ia-testip-1" name="ip" value="10.200.200.222"/>
             <nvpair id="ia-testip-2" name="broadcast" value="10.200.200.255"/>
             <nvpair id="ia-testip-3" name="nic" value="eth1"/>
             <nvpair id="ia-testip-4" name="cidr_netmask" value="24"/>
           </attributes>
         </instance_attributes>
         <operations>
           <op id="op-proftpd-1" name="monitor" interval="21s" timeout="5s" start_delay="21s"/>
         </operations>
       </primitive>
     </resources>
     <constraints>
       <rsc_order id="drbd1_before_testip" from="testip" action="start" to="ms-drbd1" to_action="promote"/>
       <rsc_colocation id="testip_on_drbd1" to="ms-drbd1" to_role="master" from="testip" score="infinity"/>
     </constraints>
   </configuration>
   <status>
     <node_state id="6676c8d5-552e-4a86-a2ea-172e954f7342" uname="dktest2sles10" crmd="online" crm-debug-origin="do_lrm_query" shutdown="0" ha="active" in_ccm="true" join="member" expected="member">
       <transient_attributes id="6676c8d5-552e-4a86-a2ea-172e954f7342">
         <instance_attributes id="status-6676c8d5-552e-4a86-a2ea-172e954f7342">
           <attributes>
             <nvpair id="status-6676c8d5-552e-4a86-a2ea-172e954f7342-pingd10220" name="pingd10220" value="750"/>
             <nvpair id="status-6676c8d5-552e-4a86-a2ea-172e954f7342-probe_complete" name="probe_complete" value="true"/>
             <nvpair id="status-6676c8d5-552e-4a86-a2ea-172e954f7342-fail-count-drbd1:0" name="fail-count-drbd1:0" value="1"/>
           </attributes>
         </instance_attributes>
         <instance_attributes id="master-6676c8d5-552e-4a86-a2ea-172e954f7342">
           <attributes>
             <nvpair id="status-master-drbd1:0-6676c8d5-552e-4a86-a2ea-172e954f7342" name="master-drbd1:0" value="10"/>
           </attributes>
         </instance_attributes>
       </transient_attributes>
       <lrm id="6676c8d5-552e-4a86-a2ea-172e954f7342">
         <lrm_resources>
           <lrm_resource id="drbd1:0" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="drbd1:0_monitor_0" operation="monitor" crm-debug-origin="build_active_RAs" transition_key="4:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition_magic="4:7;4:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call_id="2" crm_feature_set="2.1" rc_code="7" op_status="4" interval="0" last_run="96950" last_rc_change="96950" exec_time="100" queue_time="0" op_digest="b9cf2b3d762cdf1ed2eb02a10ad36f06"/>
             <lrm_rsc_op id="drbd1:0_start_0" operation="start" crm-debug-origin="build_active_RAs" transition_key="9:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition_magic="0:0;9:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call_id="4" crm_feature_set="2.1" rc_code="0" op_status="0" interval="0" last_run="95500" last_rc_change="95500" exec_time="150" queue_time="0" op_digest="b9cf2b3d762cdf1ed2eb02a10ad36f06"/>
             <lrm_rsc_op id="drbd1:0_promote_0" operation="promote" crm-debug-origin="build_active_RAs" transition_key="8:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition_magic="0:0;8:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call_id="7" crm_feature_set="2.1" rc_code="0" op_status="0" interval="0" last_run="91010" last_rc_change="91010" exec_time="130" queue_time="0" op_digest="b9cf2b3d762cdf1ed2eb02a10ad36f06"/>
             <lrm_rsc_op id="drbd1:0_monitor_10000" operation="monitor" crm-debug-origin="build_active_RAs" transition_key="9:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition_magic="4:8;9:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call_id="10" crm_feature_set="2.1" rc_code="8" op_status="4" interval="10000" last_run="8190" last_rc_change="58500" exec_time="50" queue_time="0" op_digest="e54028ea0c87b98f35b3e83c3bc800d6"/>
             <lrm_rsc_op id="drbd1:0_post_notify_stop_0" operation="notify" crm-debug-origin="build_active_RAs" transition_key="47:5:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition_magic="0:0;47:5:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call_id="13" crm_feature_set="2.1" rc_code="0" op_status="0" interval="0" last_run="52000" last_rc_change="52000" exec_time="90" queue_time="0" op_digest="b9cf2b3d762cdf1ed2eb02a10ad36f06"/>
           </lrm_resource>
           <lrm_resource id="testip" type="IPaddr2" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="testip_monitor_0" operation="monitor" crm-debug-origin="build_active_RAs" transition_key="5:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition_magic="4:7;5:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call_id="3" crm_feature_set="2.1" rc_code="7" op_status="4" interval="0" last_run="96950" last_rc_change="96950" exec_time="70" queue_time="0" op_digest="8ef566e41dbc36177896c80fa7d2176b"/>
             <lrm_rsc_op id="testip_start_0" operation="start" crm-debug-origin="build_active_RAs" transition_key="38:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition_magic="0:0;38:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call_id="9" crm_feature_set="2.1" rc_code="0" op_status="0" interval="0" last_run="88520" last_rc_change="88520" exec_time="60" queue_time="0" op_digest="8ef566e41dbc36177896c80fa7d2176b"/>
             <lrm_rsc_op id="testip_monitor_21000" operation="monitor" crm-debug-origin="build_active_RAs" transition_key="39:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition_magic="0:0;39:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call_id="11" crm_feature_set="2.1" rc_code="0" op_status="0" interval="21000" last_run="4300" last_rc_change="67420" exec_time="40" queue_time="0" op_digest="179ca11a2bb1cedf4bb653e653bd6ba9"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
     </node_state>
     <node_state id="c563f60c-45a8-439e-85dc-645f4ecc150d" uname="dktest1sles10" crmd="online" crm-debug-origin="do_lrm_query" ha="active" in_ccm="true" join="member" expected="member" shutdown="0">
       <lrm id="c563f60c-45a8-439e-85dc-645f4ecc150d">
         <lrm_resources/>
       </lrm>
     </node_state>
   </status>
 </cib>
