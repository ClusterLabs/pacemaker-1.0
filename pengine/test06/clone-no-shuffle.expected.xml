<?xml version="1.0" encoding="UTF-8"?>
<cib crm_feature_set="2.1" admin_epoch="0" epoch="5" num_updates="41" dc-uuid="6676c8d5-552e-4a86-a2ea-172e954f7342" have-quorum="1" remote-tls-port="0" validate-with="pacemaker-1.0">
   <configuration>
     <crm_config>
       <cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="dc-version" value="0.6.2-node: 7f22385d1b41c36c4cc54f292e55869caa94ab7e"/><nvpair name="no-quorum-policy" value="stop" id="nvpair"/><nvpair name="symmetric-cluster" value="true" id="nvpair"/><nvpair name="stonith-enabled" value="true" id="nvpair"/><nvpair name="stonith-action" value="reboot" id="nvpair"/><nvpair name="default-resource-stickiness" value="0" id="nvpair"/><nvpair name="default-resource-failure-stickiness" value="-INFINITY" id="nvpair"/><nvpair name="is-managed-default" value="true" id="nvpair"/><nvpair name="default-action-timeout" value="20s" id="nvpair"/><nvpair name="stop-orphan-resources" value="true" id="nvpair"/><nvpair name="stop-orphan-actions" value="true" id="nvpair"/><nvpair name="remove-after-stop" value="false" id="nvpair"/><nvpair name="startup-fencing" value="true" id="nvpair"/><nvpair name="last-lrm-refresh" id="nvpair" value="1204805697"/></cluster_property_set>
     </crm_config>
     <nodes>
       <node id="6676c8d5-552e-4a86-a2ea-172e954f7342" uname="dktest2sles10" type="normal"/>
       <node id="c563f60c-45a8-439e-85dc-645f4ecc150d" uname="dktest1sles10" type="normal"/>
     </nodes>
     <resources>
       <master id="ms-drbd1"><meta_attributes id="master_slave-ms-drbd1.meta"/><meta_attributes id="meta_attributes"><nvpair id="nvpair" name="clone-max" value="2"/><nvpair id="nvpair" name="clone-node-max" value="1"/><nvpair id="nvpair" name="master-max" value="1"/><nvpair id="nvpair" name="master-node-max" value="1"/><nvpair id="nvpair" name="notify" value="yes"/><nvpair id="nvpair" name="globally-unique" value="false"/><nvpair id="nvpair" name="target-role" value="started"/></meta_attributes><primitive id="drbd1" class="ocf" provider="heartbeat" type="drbd"><meta_attributes id="primitive-drbd1.meta"/>
           <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="drbd_resource" value="drbd1"/></instance_attributes>
           <operations>
             <op name="monitor" interval="10s" id="op-ms-drbd1-1" timeout="15s" start-delay="30s" role="Master"/>
             <op name="monitor" interval="11s" id="op-ms-drbd1-2" timeout="15s" start-delay="30s" role="Slave"/>
           </operations>
         </primitive></master>
       <primitive id="testip" class="ocf" type="IPaddr2" provider="heartbeat"><meta_attributes id="primitive-testip.meta"/>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="ip" value="10.200.200.222"/><nvpair id="nvpair" name="broadcast" value="10.200.200.255"/><nvpair id="nvpair" name="nic" value="eth1"/><nvpair id="nvpair" name="cidr_netmask" value="24"/></instance_attributes>
         <operations>
           <op name="monitor" interval="21s" id="op-proftpd-1" timeout="5s" start-delay="21s"/>
         </operations>
       </primitive>
     </resources>
     <constraints>
       <rsc_order id="drbd1_before_testip" first="ms-drbd1" then="testip" then-action="start" first-action="promote"/>
       <rsc_colocation id="testip_on_drbd1" rsc="testip" with-rsc="ms-drbd1" with-rsc-role="Master" score="INFINITY"/>
     </constraints>
   </configuration>
   <status>
     <node_state id="6676c8d5-552e-4a86-a2ea-172e954f7342" uname="dktest2sles10" crmd="online" shutdown="0" ha="active" in_ccm="true" join="member" expected="member">
       <transient_attributes id="transient_attributes.auto-1">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="pingd10220" value="750"/><nvpair id="nvpair" name="probe_complete" value="true"/><nvpair id="nvpair" name="fail-count-drbd1:0" value="1"/></instance_attributes>
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="master-drbd1:0" value="10"/></instance_attributes>
       </transient_attributes>
       <lrm id="lrm.auto-1">
         <lrm_resources id="lrm_resources">
           <lrm_resource id="drbd1:0" type="drbd" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="drbd1-0_monitor_0" operation="monitor" transition-key="4:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition-magic="4:7;4:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call-id="2" crm_feature_set="2.1" rc-code="7" op-status="4" interval="0" last_run="96950" last_rc_change="96950" exec_time="100" queue_time="0" op-digest="b9cf2b3d762cdf1ed2eb02a10ad36f06"/>
             <lrm_rsc_op id="drbd1-0_start_0" operation="start" transition-key="9:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition-magic="0:0;9:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call-id="4" crm_feature_set="2.1" rc-code="0" op-status="0" interval="0" last_run="95500" last_rc_change="95500" exec_time="150" queue_time="0" op-digest="b9cf2b3d762cdf1ed2eb02a10ad36f06"/>
             <lrm_rsc_op id="drbd1-0_promote_0" operation="promote" transition-key="8:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition-magic="0:0;8:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call-id="7" crm_feature_set="2.1" rc-code="0" op-status="0" interval="0" last_run="91010" last_rc_change="91010" exec_time="130" queue_time="0" op-digest="b9cf2b3d762cdf1ed2eb02a10ad36f06"/>
             <lrm_rsc_op id="drbd1-0_monitor_10000" operation="monitor" transition-key="9:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition-magic="4:8;9:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call-id="10" crm_feature_set="2.1" rc-code="8" op-status="4" interval="10000" last_run="8190" last_rc_change="58500" exec_time="50" queue_time="0" op-digest="e54028ea0c87b98f35b3e83c3bc800d6"/>
             <lrm_rsc_op id="drbd1-0_post_notify_stop_0" operation="notify" transition-key="47:5:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition-magic="0:0;47:5:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call-id="13" crm_feature_set="2.1" rc-code="0" op-status="0" interval="0" last_run="52000" last_rc_change="52000" exec_time="90" queue_time="0" op-digest="b9cf2b3d762cdf1ed2eb02a10ad36f06"/>
           </lrm_resource>
           <lrm_resource id="testip" type="IPaddr2" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="testip_monitor_0" operation="monitor" transition-key="5:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition-magic="4:7;5:3:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call-id="3" crm_feature_set="2.1" rc-code="7" op-status="4" interval="0" last_run="96950" last_rc_change="96950" exec_time="70" queue_time="0" op-digest="8ef566e41dbc36177896c80fa7d2176b"/>
             <lrm_rsc_op id="testip_start_0" operation="start" transition-key="38:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition-magic="0:0;38:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call-id="9" crm_feature_set="2.1" rc-code="0" op-status="0" interval="0" last_run="88520" last_rc_change="88520" exec_time="60" queue_time="0" op-digest="8ef566e41dbc36177896c80fa7d2176b"/>
             <lrm_rsc_op id="testip_monitor_21000" operation="monitor" transition-key="39:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" transition-magic="0:0;39:4:9d0bfd83-75ef-4da1-bc8d-8ee972b05fc3" call-id="11" crm_feature_set="2.1" rc-code="0" op-status="0" interval="21000" last_run="4300" last_rc_change="67420" exec_time="40" queue_time="0" op-digest="179ca11a2bb1cedf4bb653e653bd6ba9"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
     </node_state>
     <node_state id="c563f60c-45a8-439e-85dc-645f4ecc150d" uname="dktest1sles10" crmd="online" ha="active" in_ccm="true" join="member" expected="member" shutdown="0">
       <lrm id="c563f60c-45a8-439e-85dc-645f4ecc150d">
         <lrm_resources id="lrm_resources"/>
       </lrm>
     </node_state>
   </status>
 </cib>
